A01/COMP348-A1-AlexJeanFonz/src/main/java/com/alexjeanfonz/Program.java

```diff
@@ -1,12 +1,58 @@
 package com.alexjeanfonz;
 
+import java.io.BufferedReader;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.Scanner;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
 /**
  * App entrypoint
  * @author jeanrobatto
  */
 public class Program {
     
     public static void main(String[] args) {
+        PartOneTests();
+        
+        Employee[] employees = new Employee[0];
+        try {
+            employees = ReadEmployeesFromFile();
+        } catch (Exception e) {
```

Comment:
Since this catch statement is actually part of the program (i used the same generic catch in the tests but ill delete those!), I would suggest specific catches instead of a generic one. Basically thinking about the possible exceptions and catching them separately, with a custom error message for each one

Reply:
This block does not have any expected exceptions, but I used specific catches on the file reading for FileNotFound and IO exceptions.